src/bf_dpu_update.py

```diff
@@ -662,17 +692,46 @@ def _wait_task(self, task_handle, max_second=15*60, check_step=10, err_handler=N
             raise Err_Exception(Err_Num.TASK_FAILED, task_state['message'])
         return True
 
+    def validate_args(self, items):
+        if 'UserName' in items:
+            if self.username is None:
```

Comment:
if a and b:

Reply:
I'd like to keep current structure.
Since, it will make the overall structure more clear.
if 'Username' in items:
    Do some checks related to username
...
if 'FwFile' in itmes:
    Do some check realted to FW file
...

Reply:
OK